app/apps/users/views.py

```diff
@@ -17,3 +26,31 @@ def get_queryset(self):
         else:
             user = self.request.user
             return User.objects.filter(creator=user)
+
+    def create(self, request, *args, quantity, **kwargs):
+        r = requests.get(f"https://randomuser.me/api/?results={quantity}")
```

Comment:
Missing response status evaluation like raising:
```
response.raise_for_status()
```

and handling that as custom error

Reply:
Remember that API call should be optional - only for missing data